protocol/client/migrations/0001_add_messages_contacts.up.db.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS user_messages (
+id VARCHAR PRIMARY KEY NOT NULL,
+contact_id VARCHAR NOT NULL,
```

Comment:
I would add all the whisper info we have to the DB as well, signatures, timestamps, topics, etc.

Reply:
what kind of timestamps?
we store do store message timestamp. but we don't store last timestamp per chat and other things that can be useful for history retrieval.

public key derived from msg signature is stored with each message.

all topics, except one, are stored in user_contacts table (name field). private topic is hardcoded in whisper module. i want to change that though, and created a subscription for every topic.

not sure what else we can store